files/vision.tex

```diff
@@ -0,0 +1,23 @@
+\chapter{Vision}
+This chapter will give insights into the vision this project is based upon and the boundaries for what this project will focus upon.
+
+
+\section*{The big picture}
+
+The vision behind this project is to revolutionize the club management and progress monitoring inside badminton clubs. The main part where this product will be different is a smart racket that can give the club, coach and individual player a bunch of information about their badminton performance. This information can help the coaches finding weaknesses and show them where to focus their training on a specific player. 
+
+The management portion would be a web application where management staff could handle business such as economy, practice attendance and events such as parties and game scheduling.
+
+The progress monitoring will be made possible with a mobile application that can gather data over bluetooth from sensors in the smart racket. The data will not only be available on the mobile device, it will also be available in the web application, but the mobile device is required to gather the data in the first place. The club can own smart rackets for their players to use on occasions during practice. The coach then gathers this data for both the player and club to see. The player should also have the option to buy his own smart racket so that he can monitor his progress at every single practice or match.
+
+Data will bind to the players individual user and will be shareable with the coaches and club. Players can compare their progress and stats with their friends and other players from the club. It should also be possible to share training sessions and progress on social media.
+
+\graphic{0.85}{vision.png}{Illustration of the projects vision}{vision}
+
+\section*{Boundary}
+Since the vision is more than 2 man can swallow during this project duration. Boundaries are set for the specific part of the vision that we want to achieve during this project.
+
+As mentioned in the vision, the part that makes this product different is the progress monitoring through the mobile application and smart racket. Since this is the unique feature, it is also gonna be our focus point for this project.
```

Comment:
different from what?
I think the last phrase is a bit non-formal. Gonna is "slang" for "going to". Maybe rephrased to something like: "it is the main focus of this project" ?